public/app/features/annotations/annotations_srv.ts

```diff
@@ -18,6 +29,9 @@ export class AnnotationsSrv {
   alertStatesPromise: any;
   datasourcePromises: any;
 
+  /** @ngInject */
+  constructor(private templateSrv: TemplateSrv) {}
```

Comment:
You should just use the singleton instead of injecting it.
```suggestion
  constructor() {}
```